experimental/packages/opentelemetry-instrumentation-grpc/src/grpc-js/types.ts

```diff
@@ -14,29 +14,38 @@
  * limitations under the License.
  */
 
-import type * as grpcJs from '@grpc/grpc-js';
 import type { EventEmitter } from 'events';
 import type { CALL_SPAN_ENDED } from './serverUtils';
+import {
```

Comment:
I think this can just be a type-import.
```suggestion
import type {
```